src/constants.js

```diff
@@ -20,8 +20,9 @@ export default {
     { key: '-price_count', value: 'Number of prices', icon: 'mdi-tag-multiple-outline' },
   ],
   PRICE_ORDER_BY_LIST: [
-    { key: '-created', value: 'Addition date', icon: 'mdi-clock-outline' },
+    { key: 'price', value: 'Price (cheapest first)', icon: 'mdi-currency-eur' },
```

Comment:
I would put "Price: ascending order" as I have seen on several marketplace.
Also, would put another icon, as euro is not relevant outside EU. Maybe `order-numeric-ascending`, since I'm sure the descending option will come soon :)

Reply:
good idea for the icon :+1:

Reply:
I'll change to just "Price", and we can open an issue to figure out the best wording :)

Reply:
Hmm true, my bad. I rarely go to marketplaces and when I do they are in German with autotranslation to English. In German it is "Aufsteigend" and "Absteigend" which translates to "Ascending" and "Descending"